tests/tests.py

```diff
@@ -556,13 +556,23 @@ def test_deleting_on_doc_type_change_sql(self):
 
         doc['doc_type'] = 'MockArchive'
         for cls in [MockIndicatorsSql]:
-            pillow = cls.pillow()(chunk_size=0)
+            pillow = cls.pillow()(chunk_size=0, checkpoint=mock_checkpoint())
             pillow.processor(change_from_couch_row({'changes': [], 'id': '123', 'seq': 1, 'doc': doc}),
                              PillowRuntimeContext())
             with self.engine.begin() as connection:
                 rows = connection.execute(sqlalchemy.select([cls._table]))
                 self.assertEqual(rows.rowcount, 0)
 
+
+def mock_checkpoint():
+    # for some reason the testrunner chokes on these if they are not defined inline.
+    # in the future we may want to explicitly use django tests instead of regular tests
+    # if we're going to depend on a django environment.
+    from pillowtop.checkpoints.manager import PillowCheckpoint
+    from pillowtop.dao.mock import MockDocumentStore
+    return PillowCheckpoint(MockDocumentStore(), 'mock-checkpoint')
```

Comment:
not a huge fan of having to mock out the checkpoint in the tests, but the sql checkpoint api is [well-tested in pillowtop](https://github.com/dimagi/pillowtop/blob/master/pillowtop/tests/test_checkpoints.py#L28) and i didn't want to bother with switching the testrunner to bootstrap a whole django environment